hdl21/sim/data.py

```diff
@@ -391,12 +391,6 @@ def run(self, opts: Optional[vsp.SimOptions] = None) -> vsp.SimResultUnion:
         """Invoke simulation via `vlsirtools.spice`."""
         return run(self, opts=opts)
 
```

Comment:
Recommend we just leave the `_async` methods with the deprecation warnings for now, and just have them call the (non-async) `vlsir` methods. I don't *think* there are Hdl21-viai-PyPi users out there using `sim_async`, but if there are, it'll be nicer to get that warning than immediately bricking any `sim_async` calls.

Reply:
Okay, since the async backend is gone, and everything is now concurrent, I'll just let sim_async do the same thing with the deprecation warning.